pkg/reconciler/common/releases.go

```diff
@@ -85,9 +98,50 @@ func componentDir(instance v1alpha1.KComponent) string {
 	return ""
 }
 
+func componentURL(version string, instance v1alpha1.KComponent) string {
+	component := "serving"
+	urlList := []string{}
+	switch instance.(type) {
+	case *v1alpha1.KnativeServing:
+		urlList = ServingYamlNames
+	case *v1alpha1.KnativeEventing:
+		component = "eventing"
+		for _, value := range EventingYamlNames {
+			if strings.Contains(value, "%s") {
+				// Some eventing artifacts' names always contain the version with the patch number equal to 0
+				value = fmt.Sprintf(value, fmt.Sprintf("%s.0", semver.MajorMinor(sanitizeSemver(version))))
+			}
+			urlList = append(urlList, value)
+		}
+	}
+
+	// Create the comma-separated string as the URL to retrieve the manifest
+	if len(urlList) == 0 {
+		return ""
+	}
+
+	result := fmt.Sprintf(URLLinkTemplate, component,
+		version, urlList[0])
+
+	for i := 1; i < len(urlList); i++ {
+		result = fmt.Sprintf("%s,"+URLLinkTemplate, result, component,
```

Comment:
This will leave a dangling `,`.

Please use `strings.Join` to join an array of strings with a delimiter in this case.

Reply:
Done.